src/monotonic/monotonic.jl

```diff
@@ -279,6 +296,33 @@ function calcTangents(::Type{TCoeffs}, x::AbstractVector{<:Number},
     return (m, Δ)
 end
 
+function calcTangents(::Type{TCoeffs}, x::AbstractVector{<:Number},
+    y :: AbstractVector{TEl}, method::AkimaMonotonicInterpolation) where {TCoeffs, TEl}
+
+    # based on Akima (1970),
+    # "A New Method of Interpolation and Smooth Curve Fitting Based on Local Procedures",
+    # doi:10.1145/321607.321609.
+
+    n = length(x)
+    Δ = Vector{TCoeffs}(undef, n-1)
+    m = Vector{TCoeffs}(undef, n)
+    for k in 1:n-1
+        Δk = (y[k+1] - y[k]) / (x[k+1] - x[k])
+        Δ[k] = Δk
```

Comment:
```suggestion
        Δ[k] = (y[k+1] - y[k]) / (x[k+1] - x[k])
```

I think this can be a single line.

Reply:
I had the same thought when I copied the `calcTangents` from another method in order to modify it for the Akima interpolation, but I also tought there was a good reason for that since all other `calcTangents` methods do it this way!

Reply:
I don't see a reason for it. I'll change the others later, or you can. Please change this.